src/lib/menu/menu.spec.ts

```diff
@@ -1043,7 +1082,7 @@ class SimpleMenu {
 @Component({
   template: `
     <button [mdMenuTriggerFor]="menu" #triggerEl>Toggle menu</button>
-    <md-menu [xPosition]="xPosition" [yPosition]="yPosition" #menu="mdMenu">
+    <md-menu [xPosition]="xPosition" [yPosition]="yPosition"#menu="mdMenu">
```

Comment:
Missing space before `#menu`

Reply:
@willshowell  I will make the changes and raise the PR